src/video/canvas/canvas_renderer.js

```diff
@@ -253,6 +253,19 @@
             this.blitSurface();
         },
 
+
+        /**
+         * resizes the canvas
+         * @name resize
+         * @memberOf me.CanvasRenderer
+         * @function
+         */
+        resize : function (width, height)
+        {
+            this.backBufferCanvas.width = width;
+            this.backBufferCanvas.height = height;
```

Comment:
This can be moved to `me.Renderer` is you use:

```javascript
this.getCanvas().width = width;
this.getCanvas().height = height;
```

Then remove the WebGLRenderer.resize method (both classes extend `me.Renderer`)